src/percpu.rs

```diff
@@ -1,11 +1,9 @@
-use core::marker::PhantomData;
-
-use tock_registers::LocalRegisterCopy;
-
 use axerrno::{AxError, AxResult};
 use axvcpu::{AxArchPerCpu, AxVCpuHal};
+use core::marker::PhantomData;
```

Comment:
move `core::*` to the beginning of this file.

Reply:
Already done.